app/src/main/java/ch/epfl/polychef/HomePage.java

```diff
@@ -110,4 +124,81 @@ public boolean onNavigationItemSelected(@NonNull MenuItem selectedItem) {
                 }
         );
     }
+
+    protected void retrieveUserInfo(String email) {
```

Comment:
We should maybe integrate the user's related method to _ConnectedActivity_ ?
Seems like it could be better if later on we want to have a login that redirect to another connected page than _HomePage_ and these method seems to be only accessible through a _ConnectedActivity_

Reply:
Good point, we will do that during a later sprint.